tests/scripts/test-e2e-cluster.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+# Input env variables:
+# - PROVIDER - the LLM provider to be used during the test
+# - PROVIDER_KEY_PATH - path to a file containing the credentials to be used with the llm provider
+# - MODEL - name of the model to use during e2e testing
+
+# Script flow:
+# 1) Install OLS into a namespace on the cluster with valid config/api tokens and exposed via a route
+# 2) Setup a service account w/ permission to access OLS
+# 3) Wait for the ols api server to be available
+# 4) Invoke the test-e2e Makefile target
+
+
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+. "$DIR/utils.sh"
+
+# temp directory for generated resource yamls
+TMPDIR=$(mktemp -d)
+
+# ARTIFACT_DIR is defined when running in a prow job, content
+# in this location is automatically collected at the end of the test job
+ARTIFACT_DIR=${ARTIFACT_DIR:-$TMPDIR}
+
+# Deletes may fail if this is the first time running against
+# the cluster, so ignore failures
+oc delete ns openshift-lightspeed || true
+oc delete clusterrole ols-sar-check || true
+oc delete clusterrolebinding ols-sar-check || true
+oc delete clusterrole ols-user || true
+
```

Comment:
I'd suggest `oc delete --wait --ignore-not-found`. `--ignore-not-found` should make ignoring failures unnecessary and `--wait` should make the subsequent `oc create` more reliable.

Reply:
sounds good